protoc-gen-grpc-gateway/httprule/compile.go

```diff
@@ -105,10 +107,11 @@ func (t template) Compile() Template {
 		}
 	}
 	return Template{
-		Version: opcodeVersion,
-		OpCodes: ops,
-		Pool:    pool,
-		Verb:    t.verb,
-		Fields:  fields,
+		Version:  opcodeVersion,
+		OpCodes:  ops,
+		Pool:     pool,
+		Verb:     t.verb,
+		Fields:   fields,
+		Template: t.template,
```

Comment:
Yeah, really good tip. Having the template available in the swagger side will allow me to split it and iterate over it to fix those entries up.